cmd/capture.go

```diff
@@ -75,6 +77,7 @@ func init() {
 	captureCmd.Flags().BoolVarP(&libbpfOutput, "include-libbpf-output", "l", false, "Include the libbpf output")
 
 	captureCmd.Flags().BoolVarP(&save, "save", "S", false, "Save output to a file")
+	captureCmd.Flags().StringVarP(&name, "name", "n", "", "Specify a name for saved output")
```

Comment:
```suggestion
	captureCmd.Flags().StringVarP(&filename, "filename", "n", "", "Specify a filename for saved output")
```

Reply:
I want to have consistency and keep the flag "name" since other commands are using the same